gno.land/cmd/gnoweb/pages/HOME.md

```diff
@@ -1,50 +1,77 @@
-# Welcome to **Gno.land**
+# Welcome to Gno.land
 
-- [About Gno.land](/about)
+---
+
+## An interpretation of the Golang (Go) programming language for advanced developers and intrepid pioneers to build succinct, composable smart contracts for social coordination.
```

Comment:
Is this from you ? I like to be an intrepid pioneer xD

Reply:
If only I could be the writer haha! It's from Christina 😆